test/python/tile_array.py

```diff
@@ -253,9 +253,9 @@ def ipu():
         print(find_neighbors(tiles[2, 2].df))
 
         assert tiles[1:3, 1:3].neighbors.shape == (2, 2)
-        # CHECK: tile(col=1, row=1) Neighbors(north=%tile_1_2 = aie.tile(1, 2), west=None, south=None)
-        # CHECK: tile(col=1, row=2) Neighbors(north=%tile_1_3 = aie.tile(1, 3), west=%tile_0_2 = aie.tile(0, 2), south=%tile_1_1 = aie.tile(1, 1))
-        # CHECK: tile(col=2, row=1) Neighbors(north=%tile_2_2 = aie.tile(2, 2), west=%tile_1_1 = aie.tile(1, 1), south=None)
-        # CHECK: tile(col=2, row=2) Neighbors(north=%tile_2_3 = aie.tile(2, 3), west=%tile_1_2 = aie.tile(1, 2), south=%tile_2_1 = aie.tile(2, 1))
+        # CHECK: tile(col=1, row=1) : Neighbors(north=%tile_1_2 = aie.tile(1, 2), west=None, south=None)
```

Comment:
unrelated change?

Reply:
These are effectively tests (and basically my whole goal) for the feature: grab neighboring tiles, grab buffers in those tiles. Problem became (took the whole day), that the BCF linker and clang don't cooperate when it comes to data in `rodata`. Hence "add `initial_value` to `aie.buffer`".